ide-vscode.html

```diff
@@ -0,0 +1,575 @@
+
+    Make sure you have the <a href="https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-pack" target="_blank">Extension Pack for Java</a> and <a href="https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-gradle" target="_blank">Gradle for Java</a> installed, which includs the required extensions to support Gradle projects.
+</p>
+
+<h5>1. Create a Gradle project</h5>
+
+<p>
+    Select <kbd>Help -> Show All Commands</kbd>, in the dropdown list, enter <kbd>Java: Create Java project...</kbd>, and select it.
+
+    <a href="images/ide/vscode/gradle/vscode01.png" target="_blank"><img src="images/ide/vscode/gradle/vscode01.png" alt="Create a Java project"/></a>
+
+    Select <kbd>Gradle</kbd> in the list as the project type.
+
+    <a href="images/ide/vscode/gradle/vscode02.png" target="_blank"><img src="images/ide/vscode/gradle/vscode02.png" alt="Create a Gradle project"/></a>
+
+    Select the project location in the file explorer and the build script DSL. Input the project name.
+
+    Then the <kbd>Gradle for Java</kbd> extension will create a new Gradle project for you.
+
+    The new project will be opened in a new Visual Studio Code window after the creation finished.
+
+    <a href="images/ide/vscode/gradle/vscode03.png" target="_blank"><img src="images/ide/vscode/gradle/vscode03.png" alt="Open the project"/></a>
+
+    you can remove the <kbd>app/src/test</kbd> folder if you don't want to add tests for the project.
+</p>
+
+<h5>2. Modify the build</h5>
+
+<p>
+    Edit the <kbd>build.gradle</kbd> file under the <kbd>app/</kbd> folder and replace it with this
+    <a class="samples" href="https://github.com/openjfx/samples/blob/master/IDE/VSCode/Non-Modular/Gradle/hellofx/build.gradle" target="_blank">build</a> file, setting the 
+    <kbd>mainClassName</kbd> accordingly to <kbd>org.openjfx.MainApp</kbd>.
+
+    Make sure the plugin <kbd>org.openjfx.javafxplugin</kbd> is applied and the version is set to <kbd><span class="JFX_PLUGIN_VERSION">0.0.10</span></kbd>
+</p>
+<p>
+    Similar to Maven, we can declare the required JavaFX modules in the <kbd>build.gradle</kbd> file.
+    However, for Gradle we need to apply the JavaFX gradle plugin:
+</p>
+
+<pre><code>
+javafx {
+    version = "<span class="JFX_VERSION">17.0.1</span>"
+    modules = [ 'javafx.controls', 'javafx.fxml' ]
+}
+</code></pre>
+
+<p>
+    Right click on the <kbd>settings.gradle</kbd> and select <kbd>Update Project</kbd> to update the gradle project.
+
+    <a href="images/ide/vscode/gradle/vscode04.png" target="_blank"><img src="images/ide/vscode/gradle/vscode04.png" alt="Update the project"/></a>
+
+    And you will get the JavaFX dependencies after the update.
+
+    <a href="images/ide/vscode/gradle/vscode05.png" target="_blank"><img src="images/ide/vscode/gradle/vscode05.png" alt="Gradle dependencies"/></a>
+
+    As for any other Gradle dependencies, these jars can be found in the local <kbd>.gradle</kbd> repository.
```

Comment:
I don't get this statement. What other Gradle dependencies are we referring to?

Reply:
I copied this from the Eclipse tutorial, see:
- `Non-modular with Maven` > `2. Verify the pom`
- `Non-modular with Gradle` > `2. Modify the build`

I guess it intends to tell the readers that the build tool will manage all the dependencies from different projects into those folders?

I agree that this statement somehow introduces confusions and I'm ok to remove it.

Reply:
In that case I would keep these as it is and update the sentences for all IDE together in a later PR.